--- allegro-4.3.0/include/allegro/platform/almngw32.h.orig	2006-07-24 14:10:58.000000000 +0200
+++ allegro-4.3.0/include/allegro/platform/almngw32.h	2007-04-03 23:23:47.948586462 +0200
@@ -81,11 +81,6 @@
    #define HMONITOR_DECLARED 1
 #endif
 
-/* already defined... */
-#ifndef _LPCWAVEFORMATEX_DEFINED
-   #define _LPCWAVEFORMATEX_DEFINED
-#endif
-
 #endif /* ALLEGRO_SRC */
 
 /* another instance of missing constants in the mingw32 headers */
