--- allegro-4.3.0/src/win/wgdi.c.orig	2006-07-24 14:10:51.000000000 +0200
+++ allegro-4.3.0/src/win/wgdi.c	2007-04-03 22:50:52.616018674 +0200
@@ -118,7 +118,7 @@
 /* hardware mouse cursor emulation */
 static int mouse_on = FALSE;
 static int mouse_was_on = FALSE;
-static BITMAP *mouse_sprite = NULL;
+static BITMAP *wmouse_sprite = NULL;
 static BITMAP *mouse_frontbuffer = NULL;
 static BITMAP *mouse_backbuffer = NULL;
 static int mouse_xfocus, mouse_yfocus;
@@ -130,9 +130,9 @@
  */
 static int gfx_gdi_set_mouse_sprite(struct BITMAP *sprite, int xfocus, int yfocus)
 {
-   if (mouse_sprite) {
-      destroy_bitmap(mouse_sprite);
-      mouse_sprite = NULL;
+   if (wmouse_sprite) {
+      destroy_bitmap(wmouse_sprite);
+      wmouse_sprite = NULL;
 
       destroy_bitmap(mouse_frontbuffer);
       mouse_frontbuffer = NULL;
@@ -141,8 +141,8 @@
       mouse_backbuffer = NULL;
    }
 
-   mouse_sprite = create_bitmap(sprite->w, sprite->h);
-   blit(sprite, mouse_sprite, 0, 0, 0, 0, sprite->w, sprite->h);
+   wmouse_sprite = create_bitmap(sprite->w, sprite->h);
+   blit(sprite, wmouse_sprite, 0, 0, 0, 0, sprite->w, sprite->h);
 
    mouse_xfocus = xfocus;
    mouse_yfocus = yfocus;
@@ -167,7 +167,7 @@
    blit(gdi_screen, mouse_frontbuffer, x, y, 0, 0, mouse_frontbuffer->w, mouse_frontbuffer->h);
 
    /* draw the mouse pointer onto the frontbuffer */
-   draw_sprite(mouse_frontbuffer, mouse_sprite, 0, 0);
+   draw_sprite(mouse_frontbuffer, wmouse_sprite, 0, 0);
 
    hdc = GetDC(allegro_wnd);
 
@@ -312,7 +312,7 @@
 
       /* update mouse pointer if needed */
       if (mouse_on) {
-         if ((mouse_ypos+mouse_sprite->h > top_line) && (mouse_ypos <= bottom_line)) {
+         if ((mouse_ypos+wmouse_sprite->h > top_line) && (mouse_ypos <= bottom_line)) {
             blit(gdi_screen, mouse_backbuffer, mouse_xpos, mouse_ypos, 0, 0,
                  mouse_backbuffer->w, mouse_backbuffer->h);
 
@@ -540,9 +540,9 @@
    gdi_screen = NULL;
 
    /* destroy mouse bitmaps */
-   if (mouse_sprite) {
-      destroy_bitmap(mouse_sprite);
-      mouse_sprite = NULL;
+   if (wmouse_sprite) {
+      destroy_bitmap(wmouse_sprite);
+      wmouse_sprite = NULL;
 
       destroy_bitmap(mouse_frontbuffer);
       mouse_frontbuffer = NULL;
